File: app/src/main/java/net/iessochoa/joelsemperedura/practica2/AgendaConstraintActivity.java
package net.iessochoa.joelsemperedura.practica2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class AgendaConstraintActivity extends AppCompatActivity {
    Button btnOk;
    Button btnCancel;
    EditText etNombrePersona;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_agenda_constraint);

        //  Iniciamos las views
        iniciaViews();

        //Acciones de los botones
        btnCancel.setOnClickListener(view -> {
            finish();
        });

        //Lanza un mensaje flotante que hace referencia al edit text del nombre introducido para el contacto
        btnOk.setOnClickListener(view -> {
            Toast tostada= Toast.makeText(getApplicationContext(),etNombrePersona.getText().toString(), Toast.LENGTH_SHORT);
            tostada.show();
        });

    }


    private void iniciaViews(){
        btnOk=findViewById(R.id.btnOk);
        btnCancel=findViewById(R.id.btnCancel);
        etNombrePersona=findViewById(R.id.etNombrePersona);
    }
}

File: app/src/main/java/net/iessochoa/joelsemperedura/practica2/AgendaConstraintActivity.kt
package net.iessochoa.joelsemperedura.practica2

import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class AgendaConstraintActivity : AppCompatActivity() {
    lateinit var btnOk: Button
    lateinit var btnCancel: Button
    lateinit  var etNombrePersona: EditText

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_agenda_constraint)

        //***Inicia las views***//
        iniciaViews()

        //******BOTONES******//

        //***Termina la activity***//
        btnCancel.setOnClickListener { finish() }

        //***Mensaje tipo Toast***//
        btnOk.setOnClickListener {
            val toast = Toast.makeText( applicationContext,
                etNombrePersona.text.toString(), Toast.LENGTH_SHORT
            )
            toast.show()
        }
    }
    //***Inicia las views***//
    private fun iniciaViews() {
        btnOk = findViewById(R.id.btnOk)
        btnCancel = findViewById(R.id.btnCancel)
        etNombrePersona = findViewById(R.id.etNombrePersona)
    }

}